docs/_data/opensource/aws/ami/ami_ids_4.6.10.csv

```diff
@@ -0,0 +1,23 @@
+Region,Region Code,Ami_id_x86_64,Ami_id_aarch64,Cluster_x86_64,Cluster_aarch64

```

Comment:
whenever i see user starts tp use git as a database, it worries me. in general, the source of the database is another database. and this database is but a processed version. this has some sequences:

1. the processed version is likely to be outdated. and should be updated manually. 
2. the connection between these two database is implicit.  for instance, we don't use a shared version number or a timestamp here.
3. the delta between the updates is much less interesting than the regular ones.
4. we use source/version control system to track data.

Reply:
i admit that the comments above are not constructive. now that the source of the CSV files is https://downloads.scylladb.com/downloads/scylla/aws/ami/ , how about integrating the process of updating the document as a part of the process updating the AMI images for a certain major release? so, for instance, when we release 5.4, we not only upload a new AMI, we also trigger the doc build process to update the document.

i also am wondering, as a dictionary of the published AMI images, shall we update the document of all versions when we publish a new AMI for any major version, or shall we update it only when a new release of given version is published?